src/TaskRunner/Commands/ComponentCheckCommands.php

```diff
@@ -232,10 +232,6 @@ protected function printComponentResults(ConsoleIO $io)
      */
     protected function validateComponent(array $package, array $modules)
     {
-        // Only validate module components for this time.
-        if (!isset($package['type']) || $package['type'] !== 'drupal-module') {
```

Comment:
Please include in admin/config/qa/qa_endpoints/toolkit-settings a setting to control the type of package that will be checked by this task.

By default "drupal-module" on.